test/conformance/tests/sriov_operator.go

```diff
@@ -379,15 +119,14 @@ var _ = Describe("[sriov] operator", func() {
 				validationFunction([]string{sriovNetwork.Name}, checkFunc)
 			}
 
-			BeforeEach(func() {
-				var err error
+			execute.BeforeAll(func() {
```

Comment:
Can we put this on the outer level? It will be more clear that the same policy is shared across all the tests

Reply:
Done